src/app/features/search/multiple-search-results/multiple-search-results.component.spec.ts

```diff
@@ -11,20 +18,88 @@ describe('MultipleSearchResultsComponent', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [MultipleSearchResultsComponent],
-      imports: [HttpClientTestingModule],
+      declarations: [SingleSearchResultComponent, MultipleSearchResultsComponent],
+      imports: [DynamicFormsModule, HttpClientTestingModule, RouterTestingModule],
       providers: [provideMockStore({ initialState: initialAppState })]
     }).compileComponents();
   });
 
-  beforeEach(() => {
-    store = TestBed.inject(MockStore);
-    fixture = TestBed.createComponent(MultipleSearchResultsComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
+  describe('default tests', () => {
+    beforeEach(() => {
+      store = TestBed.inject(MockStore);
+      store.overrideSelector(selectQueryParams, { vin: '123456' });
+
+      fixture = TestBed.createComponent(MultipleSearchResultsComponent);
+      component = fixture.componentInstance;
+
+      fixture.detectChanges();
+    });
+
+    it('should create', () => {
+      expect(component).toBeTruthy();
+    });
+
+    it('should navigate back', fakeAsync(() => {
+      const navigateBackSpy = jest.spyOn(component, 'navigateBack');
+
+      const button = fixture.debugElement.query(By.css('.govuk-back-link'));
+      expect(button).toBeTruthy();
+      (button.nativeElement as HTMLButtonElement).click()
+
+      tick();
+
+      expect(navigateBackSpy).toHaveBeenCalled();
+    }));
   });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  describe('searching', () => {
+    beforeEach(() => {
+      store = TestBed.inject(MockStore);
+      store.overrideSelector(vehicleTechRecords, mockVehicleTechnicalRecordList());
+    });
+
+    it('should search using a vin', () => {
+      store.overrideSelector(selectQueryParams, { vin: '123456' });
+
+      fixture = TestBed.createComponent(MultipleSearchResultsComponent);
+      component = fixture.componentInstance;
+
+      fixture.detectChanges();
+
+      component.vehicleTechRecords$.subscribe(record => expect(record).toBeTruthy());
```

Comment:
Yes, and I can confirm this in 2 ways:
1. I have tested these manually.
2. The code coverage is proof of this. The lines 23-31 in "[src/app/features/search/multiple-search-results/multiple-search-results.component.ts](https://github.com/dvsa/cvs-app-vtm/pull/394/files#diff-831420027b8cc919ca8c3cd57851985515c4a81eb01391c96b1f53f44e9c3f89)" would not have been covered otherwise.